Rename Monitoring bundle to Observability plugin. (#2269)
knative/docs: docs/install/any-kubernetes-cluster.md

You can find a number of samples for Knative Eventing [here](../eventing/samples/README.md). A quick-start guide is available [here](../eventing/getting-started.md).

## Installing the Observability plugin

{{< feature-state version="v0.2" state="alpha" >}}

> Knative provides a collection of observability components that can be used to make the Serving and Eventing components more observable.

Reviewer: ```suggestion
Install the following observability features to enable logging, metrics, and request tracing in your Serving and Eventing components.
```

> Before you install any specific observability components, you must first install the core:

Reviewer: ```suggestion
 All observibility plugins require that you first install the core:
```

```bash

> kubectl apply --filename {{< artifact repo="serving" file="monitoring-core.yaml" >}}

Reviewer: Can we add below this:

"After the core is installed, you can choose to install one or all of the following observability plugins:"
